_docs/delegators.md

```diff
@@ -0,0 +1,78 @@
+
+## How to stake
+
+### Step 1: Figure out who you want to stake to
+
+Before staking tokens to collators, we recommend you do your research on who to stake towards. To figure out the candidate pool, you can find the set of collators in [Subscan](https://turing.subscan.io/validator). Save their wallet address as `COLLATOR_WALLET_ADDRESS` to be used on the step below.
+
+### Step 2: Figure out your inputs
+To be stake your delegator via PolakdotJS extrinsics, you'll need to figure out three numbers. We've provided a helpful script for you to use. Navigate to the **Developer > Javascript** tab on the [PolkadotJS App](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/js).
+
+```javascript
+const collatorWalletAddress = 'COLLATOR_WALLET_ADDRESS';
+const delegatorWalletAddress = 'YOUR_WALLET_ADDRESS';
+const minDelegationStake = await api.consts.parachainStaking.minDelegation;
+const candidateInfo = await api.query.parachainStaking.candidateInfo(collatorWalletAddress);
+const candidateDelegationCount = JSON.parse(candidateInfo).delegationCount;
+const delegatorState = await api.query.parachainStaking.delegatorState(delegatorWalletAddress);
+const delegationsLength = delegatorState.delegations ? delegatorState.delegations.length : 0;
+console.log(`2a. Minimum amount to be staked: ${minDelegationStake}`);
+console.log(`2b. Candidate delegation count: ${candidateDelegationCount}`);
+console.log(`2c. Delegation count: ${delegationsLength}`);
+```
+
+### Step 3: Stake your tokens to a collator
+
+Currently, everything related to staking needs to be accessed via the Extrinsics menu, under the Developer tab. To delegate a candidate, provide the following information:
+
+![delegate](../../assets/img/staking-delegation/delegate.png)
+
+- `candidate`: This will be the collator you so choose to stake towards from your research in Step 1 and has the same `COLLATOR_WALLET_ADDRESS` from Step 2 above.
+- `amount`: Figure out how much you would like to stake to the collator. At minimum you must stake `2a. Minimum amount to be staked` from the output above.
+- `candidateDelegationCount`: Values ​​are available from section above as `2b. Candidate delegation count`.
+- `delegationCount`: Values ​​are available from the section above as `2c. Delegation count`.
+
+---
+## How to stop staking to a collator
+
+### Step 1: Schedule Request to Stop Delegations
+
+To delegate a candidate, send `parachainStaking.scheduleRevokeDelegation` extrinsic by the following information:
+
+![schedule-revoke-delegation](../../assets/img/staking-delegation/schedule-revoke-delegation.png)
+
+Once you have scheduled an exit delay(24 rounds * 600 block, 12 seconds per block in current Turing Network), you must wait an exit delay before you can then execute it.
```

Comment:
Probably worth being explicit that funds don't actually unlock until you perform the second extrinsic to execute the exit.

Reply:
I'm gonna remove this for now. I wanna test first